source/functions/mongodb/write.txt

```diff
@@ -248,7 +248,7 @@ an array field:
    .. code-block:: json
       
       { "_id" : 1, "grades" : [ 85, 82, 80 ] }
-      { "_id" : 2, "grades" : [ 88, 90, 92 ] }
+      { "_id" : 2, "grades" : [ 89, 90, 92 ] }
```

Comment:
```suggestion
      { "_id" : 2, "grades" : [ 88, 90, 92 ] }
```
doesn't really matter, but this keeps the value aligned in subsequent examples (i.e. that the first value ends in an 8)